things/api/things/http/endpoint_test.go

```diff
@@ -584,28 +584,28 @@ func TestViewThing(t *testing.T) {
 			id:     strconv.FormatUint(wrongID, 10),
 			auth:   token,
 			status: http.StatusNotFound,
-			res:    "",
+			res:    toJSON(errorRes{things.ErrNotFound.Error()}),
 		},
 		{
 			desc:   "view thing by passing invalid token",
 			id:     sth.ID,
 			auth:   wrongValue,
 			status: http.StatusForbidden,
-			res:    "",
+			res:    toJSON(errorRes{things.ErrUnauthorizedAccess.Error()}),
```

Comment:
Same here

Reply:
also var for this, and in other places - tests in users